qml/js/yleApi.js

```diff
@@ -1,215 +1,240 @@
 .import "http.js" as HTTP

```

Comment:
I would like to try to make this `.pragma library` since it might possible free up some memory and then the API would be instantiated only once for the whole QML engine, currently it requires the globals from the global context so did not work as is. I'll get back to this at some point. :)

Reply:
I have thought whether it would make sense to move the API handling to c++ side, it could be quite efficient.